lib/bitcoin/BitcoinClient.ts

```diff
@@ -313,6 +325,128 @@ export default class BitcoinClient {
     return transaction;
   }
 
+  // @ts-ignore

```

Comment:
I assume these ts-ignores are here because they are unused right now but will be used in the future?

Reply:
Correct.